build.gradle

```diff
@@ -82,9 +82,37 @@ subprojects { subproject ->
     		}
     	}
         implementation('org.eclipse.platform:org.eclipse.core.commands') {
-                version { strictly '(,3.9.800)' }
-                because 'Versions 3.10.0 and later of this artifact require Java 11.'
+			version { strictly '(,3.9.800)' }
+			because 'Versions 3.10.0 and later of this artifact require Java 11.'
         }
+	    implementation('org.eclipse.platform:org.eclipse.e4.ui.workbench') {
+			version { strictly '(,1.11.0]' }
+			because 'Later versions of this artifact require Java 11.'
+	    }
+	    implementation('org.eclipse.platform:org.eclipse.equinox.registry') {
+			version { strictly '(,3.8.600]' }
+			because 'Dependencies from later versions introduce inconsistencies with constraints above.'
```

Comment:
I did my best to document these.  The real issue is that the Gradle constraint solver isn't "smart" enough, and greedily chooses the most recent available version of a dep consistent with the given dynamic range.  E.g., if you have dep A fixed at version X, A depends on B with some range, and some transitive dep C of B requires dep A at version X+1, constraint resolution fails.  Ideally Gradle would just choose an earlier version of C, which would be consistent with all constraints and avoid the failure, but it doesn't.  So we need to add a bunch of version restrictions.